pilot/pkg/networking/plugin/authz/rbac.go

```diff
@@ -39,22 +39,100 @@ import (
 )
 
 const (
-	// RbacFilterName is the name of the RBAC filter in envoy.
-	RbacFilterName = "envoy.filters.http.rbac"
-
-	sourceIP        = "source.ip"
-	sourceService   = "source.service"
-	destinationIP   = "destination.ip"
-	destinationPort = "destination.port"
-	userHeader      = ":user"
-	methodHeader    = ":method"
-	pathHeader      = ":path"
-	//TODO(yangminzhu): Update the key after the final header name of service is decided.
-	serviceHeader       = ":service"
-	requestHeaderPrefix = "request.header["
-	requestHeaderSuffix = "]"
+	// rbacFilterName is the name of the RBAC filter in envoy.
+	rbacFilterName = "envoy.filters.http.rbac"
+
+	// attributes that could be used in a ServiceRoleBinding property.
+	attrSrcIP     = "source.ip"     // supports both single ip and cidr, e.g. "10.1.2.3" or "10.1.0.0/16".
+	attrSrcHeader = "source.header" // header name is surrounded by brackets, e.g. "source.header[User-Agent]".
+
+	// attributes that could be used in a ServiceRole constraint.
+	attrDestIP        = "destination.ip"        // supports both single ip and cidr, e.g. "10.1.2.3" or "10.1.0.0/16".
+	attrDestPort      = "destination.port"      // must be in the range [0, 65535].
+	attrDestHeader    = "destination.header"    // header name is surrounded by brackets, e.g. "destination.header[User-Agent]".
+	attrDestLabel     = "destination.label"     // label name is surrounded by brackets, e.g. "destination.label[version]".
```

Comment:
destination.labels (missing 's' at the end).

Reply:
Done.